pwnlib/commandline/shellcraft.py

```diff
@@ -366,15 +356,15 @@ def main(args):
         sys.exit(0)
 
     if args.format in ['s', 'str', 'string']:
-        code = _string(code)
+        code = hexstr(code)
     elif args.format == 'c':
         code = '{' + ', '.join(map(hex, bytearray(code))) + '}' + '\n'
     elif args.format in ['h', 'hex']:
         code = pwnlib.util.fiddling.enhex(code) + '\n'
     elif args.format in ['i', 'hexii']:
         code = hexii(code) + '\n'
     elif args.format in ['d', 'escaped']:
-        code = ''.join('\\x%02x' % c for c in bytearray(code)) + '\n'
+        code = hexstr(code, force=True) + '\n'
```

Comment:
Functional change: quotes.

```suggestion
        code = ''.join(fr'\x{c:02x}' for c in code) + '\n'
```